vueui/src/components/view/machinery/atmospherics/freezer.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <vui-group>
+    <vui-group-item label="Status:">
+      <vui-button v-bind:class="{selected: on}" :params="{ toggleStatus: 1 }">On</vui-button>
+      <vui-button v-bind:class="{selected: !on}" :params="{ toggleStatus: 1 }">Off</vui-button>
```

Comment:
Please decide do you want to use v-bind shorthand or do not. One same hand you use full sintax and shorthand. `v-bind:class` is same as `:class`